trust/persist.h

```diff
@@ -36,8 +36,10 @@
 #define P11_PERSIST_H_
 
 #include "array.h"
+#include "buffer.h"
```

Comment:
I would put these includes into the source file

Reply:
"buffer.h" needs to be included, as `p11_persist_write` takes a `p11_buffer` argument.